packages/osd-test/src/legacy_opensearch/opensearch_test_config.js

```diff
@@ -37,11 +37,11 @@ export const opensearchTestConfig = new (class OpenSearchTestConfig {
   }
 
   getPort() {
-    return this.getUrlParts().port;
+    return this.getURL().port;
   }
 
   getUrl() {
-    return this.getUrlParts().toString();
+    return this.getURL().toString();
```

Comment:
Out of scope for this PR, but the naming of this method is unfortunate.

Reply:
What kind of naming do you prefer? If this change is out of scope, could we still use `getUrlParts`?